dsl/webflux/src/test/java/org/springframework/integration/samples/dsl/webflux/WebfluxApplicationTests.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2014-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.samples.dsl.webflux;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * @author Dietrich Schulten
+ * @since 5.0.4
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class WebfluxApplicationTests {
+
+	@Test
+	public void contextLoads() {
+	}
```

Comment:
Would be great to have some content in this test to be sure that our flow is correct.
You can borrow some ideas how to test it from here: https://github.com/spring-projects/spring-integration/blob/master/spring-integration-webflux/src/test/java/org/springframework/integration/webflux/dsl/WebFluxDslTests.java

Although you can do it with the `webEnvironment = WebEnvironment.RANDOM_PORT` and `WebTestClient.bindToServer()`.